aws_lambda.py

```diff
@@ -423,6 +438,9 @@ def sync_to_s3(target_dir, aws_region=AWS_REGION, bucket_name=BUCKET_NAME):
     context = None
     try:
         function = globals()[sys.argv[1]]
+    except IndexError as e:
+        help(None, None)
```

Comment:
I don't understand why `help` requires `event` and `context` if they aren't used.